pywps/app/Process.py

```diff
@@ -471,9 +471,9 @@ def setup_outputs_from_wps_request(self, wps_request):
                 for outpt in self.outputs:
                     if outpt.identifier == wps_outpt:
                         outpt.as_reference = is_reference
-                        if isinstance(outpt, ComplexOutput) and mimetype != '':
+                        if isinstance(outpt, ComplexOutput) and mimetype is not None:
```

Comment:
You could also simply check for mimetype, no? If it isn't `None` or `""`, it should evaluate to `True`.

Reply:
Yes ... done.